sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/ServiceClient.cs

```diff
@@ -155,6 +153,56 @@ public ServiceClient(TextAnalyticsClientOptions options)
 
         #endregion
 
+        #region Dynamic Classify
+
+        public virtual Response<ClassificationCategoryCollection> DynamicClassify(
+            string document,
+            IEnumerable<string> categories,
+            string language = default,
+            DynamicClassifyOptions options = default,
+            CancellationToken cancellationToken = default) =>
+            throw Validation.NotSupported($"{nameof(TextAnalyticsClient)}.{nameof(TextAnalyticsClient.DynamicClassify)}", ServiceVersion.V2022_10_01_Preview, ServiceVersion);
+
+        public virtual Response<DynamicClassifyDocumentResultCollection> DynamicClassifyBatch(
```

Comment:
The difference is between the LRO features (e.g. `StartAnalyzeHealthcareEntities`/`StartAnalyzeHealthcareEntitiesAsync`) and the non-LRO features (e.g., `ExtractKeyPhrases`/`ExtractKeyPhrasesAsync` and `ExtractKeyPhrasesBatch`/`ExtractKeyPhrasesBatchAsync`). Dynamic classification is the latter, although I can see it being a little confusing because custom classification is the former.